src/dmd/root/filename.d

```diff
@@ -883,11 +882,12 @@ nothrow:
         }
 
         version (Windows)
-            const r = _mkdir(path.toDString);
+            const r = _mkdir(path);
```

Comment:
path.ptr ? 0 termination?

Reply:
path is a slice.  _mkdir relies on the input's .length, not strlen